tests/functional/policies/check_triggers/w13.te

```diff
@@ -0,0 +1,13 @@
+policy_module(w13, 1.0)
+
+type foo_t;
+
+# do not warn about these
+allow foo_t foo_t:file *;
+allow foo_t foo_t:file ~map;
+dontaudit foo_t foo_t:file audit_access;
+neverallow foo_t foo_t:file audit_access;
+
+# warn about these
+allow foo_t foo_t:file audit_access;
```

Comment:
Just for the sake of thoroughness, maybe add a check for another object class, such as dir as well?

Reply:
done